schema/cognite_config.schema.json

```diff
@@ -104,6 +104,59 @@
         "external-id-prefix": {
             "type": "string",
             "description": "Prefix on externalId used when creating CDF resources"
+        },
+        "global-config": {
+            "type": "object",
+            "description": "Configure authentication using OIDC tokens",
+            "properties": {
+                "disable-gzip": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Whether or not to disable gzipping of json bodies."
+                },
+                "disable-pypi-version-check": {
```

Comment:
This field isn't actually included in config, and shouldn't be.

Reply:
Thanks for noticing, I'd added it before I noticed you were already setting that to a static `True` 👍